tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/runconfig/application/BallerinaApplicationRunningState.java

```diff
@@ -67,18 +68,36 @@ public void startNotified(ProcessEvent event) {
 
     @Override
     protected BallerinaExecutor patchExecutor(@NotNull BallerinaExecutor executor) throws ExecutionException {
-        RunConfigurationKind kind = getConfiguration().getRunKind();
+        // RunConfigurationKind kind = getConfiguration().getRunKind();
         Project project = myConfiguration.getProject();
         VirtualFile baseDir = project.getBaseDir();
-        String filePath = myConfiguration.getPackage();
+        String filePath;
 
         // Find the file in the project. This is needed to find the module. Otherwise if the file is in a sub-module
         // and the SDK for the project is not set, SDK home path will be null.
         PsiFile file = BallerinaPsiImplUtil.findFileInProject(project, myConfiguration.getFilePath());
-        Module module = null;
-        if (file != null) {
-            module = ModuleUtilCore.findModuleForPsiElement(file);
+
+        assert file != null;
```

Comment:
Remove `assert` and use an `if` condition.

Reply:
Fixed

Reply:
Done